babel.config.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
See https://github.com/silverstripe/silverstripe-asset-admin/pull/1320#discussion_r1068695104 for explanation